src/gen/relationalai/protocol/message_pb.jl

```diff
@@ -72,3 +72,8 @@ function PB._encoded_size(x::MetadataInfo)
     !isempty(x.relations) && (encoded_size += PB._encoded_size(x.relations, 1))
     return encoded_size
 end
+
+# Pretty printer to prevent large amounts of Metadata from being printed.
+function Base.show(io::IO, x::MetadataInfo)
+    print(io, "MetadataInfo (length=", length(x.relations), ")")
+end
```

Comment:
Thanks for the PR, @msagarpatel!

In the past, I believe there has been a push to keep the 2-arg show function always producing _valid julia code_ that would reconstruct the object.

For pretty-printing at the REPL, we're meant to overload 3-arg show:
```suggestion
# Pretty printer to prevent large amounts of Metadata from being printed.
function Base.show(io::IO, ::MIME"text/plain", x::MetadataInfo)
    print(io, "MetadataInfo (length=", length(x.relations), ")")
end
```

I tried to write up some guidance about that here -- we should probably publish this in the julia docs somewhere:
https://github.com/RelationalAI/raicode/issues/1211#issuecomment-583393459

Reply:
Changed applied 👍 Thanks, @NHDaly! Funnily enough, I knew about the 3-arg show method, but forgot about it: https://github.com/RelationalAI/raicode/pull/12461. Thanks for the pointer to your comment: I wish I saw it several months ago!